Created bad_team_onboarding.md (#307)
DXHeroes/knowledge-base-content: practices/bad_team_onboarding.md

---
slug: agile-events
stages:
  - development
short_description: The bad team onboarding is when your new members don't receive the crucial information or vice-versa i.e. they are overflooded with the pointers and information at the pace.
tags:
  - retrospective
  - refinement
  - demotivated team
  - less productivity
keywords:
  - demotivated team
  - retrospective
  - meaningless work
---

# Bad Team Onboarding

**TL;DR**

> A great onboarding process should help an employee identify with the company and feel a sense of belonging–like joining a family that welcomes you with open arms, but when this feeling is lost somewhere it leds to Bad Team Onboarding

Reviewer: Hello, this sentence is too similar to this one: _A great onboarding process should help an employee identify with the company and feel a sense of belonging–like joining a family that welcomes you with open arms._  from the https://www.jobvite.com/wp-content/uploads/2016/08/Jobvite_eBook_Onboarding_Gone_Wrong.pdf. 

Please, rewrite it with your own words.

## What Is Bad Team Onboarding

> Bad Team Onboarding refers to ruining entire experience of hiring, welcoming, orienting, and engaging a new hire and helping them become acclimated and integrated into the corporate culture.

Reviewer: ```suggestion
Bad Team Onboarding refers to ruining the entire experience of hiring, welcoming, orienting, and engaging with the new hire. A bad first experience can alienate new hires from the team and the team culture.
```

![Onboarding](/files/bad_team_onboarding.jpg)

> ## Why Bad Team Onboarding Occured

Reviewer: ```suggestion
## Reasons for Bad Team Onboarding
```

1. NO PLAN, NO OBJECTIVES
2. IT MOVES TOO FAST
3. INADEQUATE NEW HIRE TRAINING
4. NO FOLLOW UP, SUPPORT OR PERFORMANCE MEASUREMENT

## How to Avoid Bad Team Onboarding

> - Provide your new hire with a transparent and open explanation of the organizational structure of the organization. 

Reviewer: Hello, please, rewrite this sentence -> its too similar to _Provide your new hire with a transparent and open explanation of the organizational structure of the organization_ from https://talentculture.com/bad-hire-or-bad-onboarding-10-tips-for-successful-new-hire-assimilation/

> - Talk about opportunities that your team members have and that have been provided in the past.

Reviewer: This is also too similar to _Talk about opportunities that your team members have and that have been provided in the past_ from https://talentculture.com/bad-hire-or-bad-onboarding-10-tips-for-successful-new-hire-assimilation/

> - Have an open one-on-one meeting with your new team member. 

Reviewer: I would suggest revisiting this: _Have an open one-on-one meeting with your new team member_ similarity from https://talentculture.com/bad-hire-or-bad-onboarding-10-tips-for-successful-new-hire-assimilation/

> - Employees do not need to spend time in a room filling out paperwork for their benefits enrollment, their tax forms, etc.

Reviewer: give this a second visit, please. It seems like it was taken out of https://talentculture.com/bad-hire-or-bad-onboarding-10-tips-for-successful-new-hire-assimilation/ (_Employees do not need to spend time in a room filling out paperwork for their benefits enrollment, their tax forms, etc._)

> - Give a full tour so that a new hire knows where everything is. Sometimes, comfort level is heightened beyond what we know when we spend time on the minor things that we take for granted.

Reviewer: Hi, this also is too similar to _Give a full tour so that a new hire knows where everything is. Sometimes, comfort level is heightened beyond what we know when we spend time on the minor things that we take for granted_ from the talent culture article

> - All team members must be welcoming, friendly, and genuinely happy to have this new member on board. A special lunch or happy hour is called for.

Reviewer: Please, consider rewriting this since it seems to similar to _All team members must be welcoming, friendly, and genuinely happy to have this new member on board. A special lunch or happy hour is called for_ from https://talentculture.com/bad-hire-or-bad-onboarding-10-tips-for-successful-new-hire-assimilation/

> ## Common Pitfalls of Bad Team Onboarding

Reviewer: I like this part, however, problems don't have Pitfalls in their structure. I would suggest adding it to the ## What Is Bad Team Onboarding
